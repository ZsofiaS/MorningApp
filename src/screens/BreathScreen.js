import React from 'react';
import { View, Text, StyleSheet, Image } from 'react-native';
import Timer from '../components/Timer';
import colours from '../../constants/colours.js';

export default function BreathScreen() {
  return (
    <View>
      <Text
        style={styles.title}>
        Focus on your breathing
      </Text>
      <View style={styles.imageContainer}>
        <Image
          style={styles.image}
          source={require('../../images/lotus-position-2.png')}
        />
      </View>
      <Timer />
    </View>
  )
}

const styles = StyleSheet.create({
  title: {
    color: colours.primaryColour,
    fontSize: 25,
    marginVertical: 60,
    marginHorizontal: 50,
    textAlign: 'center'
  },
  imageContainer: {
    flex: 1,
    alignItems: 'center',
    marginVertical: 15
  },
  image: {
    width: 120,
    height: 120
  }
})
