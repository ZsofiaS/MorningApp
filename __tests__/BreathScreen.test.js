import React from 'react';
import renderer from 'react-test-renderer';
import BreathScreen from '../src/screens/BreathScreen';

test('breathscreen renders correctly', () => {
  const tree = renderer.create(<BreathScreen />).toJSON();
  expect(tree).toMatchSnapshot();
})
