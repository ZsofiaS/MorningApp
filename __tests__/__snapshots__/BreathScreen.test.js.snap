// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breathscreen renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#c48b9f",
        "fontSize": 20,
        "marginHorizontal": 50,
        "marginVertical": 60,
        "textAlign": "center",
      }
    }
  >
    Focus on your breathing
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "marginVertical": 15,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 120,
          "width": 120,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginVertical": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#c48b9f",
          "fontSize": 80,
          "marginVertical": 120,
          "textAlign": "center",
        }
      }
    >
      5
      :
      00
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginVertical": -100,
        }
      }
    >
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`homescreen renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#c48b9f",
        "fontSize": 20,
        "marginHorizontal": 50,
        "marginVertical": 60,
        "textAlign": "center",
      }
    }
  >
    Focus on your breathing
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "marginVertical": 15,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 120,
          "width": 120,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginVertical": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#c48b9f",
          "fontSize": 80,
          "marginVertical": 120,
          "textAlign": "center",
        }
      }
    >
      5
      :
      00
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginVertical": -100,
        }
      }
    >
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginHorizontal": 10,
            "width": "15%",
          }
        }
      >
        <View
          style={
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            }
          }
        >
          <View
            accessibilityRole="button"
            accessibilityState={
              Object {
                "busy": false,
                "disabled": false,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 15,
                }
              }
            >
              <Text />
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
